.github/workflows/release.yml

```diff
@@ -35,9 +35,10 @@ jobs:
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v3
         with:
-          version: latest
+          version: 'v1.18.2'
```

Comment:
Latest version (at the time of writing) is v1.19.2 and it needs extra keys. The changes are incompatible with older versions of goreleaser and goreleaser 1.19.2 requires go 1.20 so the ci breaks. So, for now I have set the goreleaser version to v1.18.2

Will take up the go version update separately and change this back to latest.